python/graphstorm/config/argument.py

```diff
@@ -747,6 +748,142 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    def _check_distill_feats(self, dt_feats):
```

Comment:
it seems you define some format for the key of dt_feats. you should explain it in the docstring.

Reply:
will do

Reply:
will do